components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -555,20 +571,28 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
             <div className="flex">
               <div className="flex w-full gap-1 items-center">
                 <Tooltip direction="top" content="Pick a date">
-                  <Popover>
+                  <Popover open={createPopoverOpen}>
                     <PopoverTrigger asChild>
-                      <button className="flex items-center gap-2 p-2 text-base rounded-full text-light-slate-9 bg-light-slate-3">
+                      <div
```

Comment:
refactor (blocking): this needs to stay as a button so that it can receive focus.